lib/upnp/upnp.go

```diff
@@ -82,6 +84,18 @@ func (e *UnsupportedDeviceTypeError) Error() string {
 	return fmt.Sprintf("Unsupported UPnP device of type %s", e.deviceType)
 }
 
+const UrnIgdV1 = "urn:schemas-upnp-org:device:InternetGatewayDevice:1"
+const UrnIgdV2 = "urn:schemas-upnp-org:device:InternetGatewayDevice:2"
+const UrnWANDeviceV1 = "urn:schemas-upnp-org:device:WANDevice:1"
+const UrnWANDeviceV2 = "urn:schemas-upnp-org:device:WANDevice:2"
+const UrnWANConnectionDeviceV1 = "urn:schemas-upnp-org:device:WANConnectionDevice:1"
+const UrnWANConnectionDeviceV2 = "urn:schemas-upnp-org:device:WANConnectionDevice:2"
+const UrnWANIPConnectionV1 = "urn:schemas-upnp-org:service:WANIPConnection:1"
+const UrnWANIPConnectionV2 = "urn:schemas-upnp-org:service:WANIPConnection:2"
+const UrnWANIPv6FirewallControlV1 = "urn:schemas-upnp-org:service:WANIPv6FirewallControl:1"
+const UrnWANPPPConnectionV1 = "urn:schemas-upnp-org:service:WANPPPConnection:1"
+const UrnWANPPPConnectionV2 = "urn:schemas-upnp-org:service:WANPPPConnection:2"
```

Comment:
These should be unexported, only for local use.  Putting them into a `const ( ... )` block makes for much nicer formatting / alignment.  Don't forget to adjust the places where they are used (start with lowercase as well).  And the one use in `igd_service.go` can be replaced as well, see earlier comment.

```suggestion
const (
	urnIgdV1                    = "urn:schemas-upnp-org:device:InternetGatewayDevice:1"
	urnIgdV2                    = "urn:schemas-upnp-org:device:InternetGatewayDevice:2"
	urnWANDeviceV1              = "urn:schemas-upnp-org:device:WANDevice:1"
	urnWANDeviceV2              = "urn:schemas-upnp-org:device:WANDevice:2"
	urnWANConnectionDeviceV1    = "urn:schemas-upnp-org:device:WANConnectionDevice:1"
	urnWANConnectionDeviceV2    = "urn:schemas-upnp-org:device:WANConnectionDevice:2"
	urnWANIPConnectionV1        = "urn:schemas-upnp-org:service:WANIPConnection:1"
	urnWANIPConnectionV2        = "urn:schemas-upnp-org:service:WANIPConnection:2"
	urnWANIPv6FirewallControlV1 = "urn:schemas-upnp-org:service:WANIPv6FirewallControl:1"
	urnWANPPPConnectionV1       = "urn:schemas-upnp-org:service:WANPPPConnection:1"
	urnWANPPPConnectionV2       = "urn:schemas-upnp-org:service:WANPPPConnection:2"
)
```

Reply:
Done